packages/property-projector/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "app-name",
+  "version": "0.0.1",
+  "private": true,
+  "description": "Description",
+  "repository": {
+    "type": "git",
+    "url": "git+git@github.com:reapit/foundations.git"
+  },
+  "license": "MIT",
+  "author": "Author",
+  "main": "./src/index.ts",
+  "scripts": {
+    "build:prod": "rimraf public/dist && webpack --config ../../scripts/webpack/webpack.config.prod.js",
+    "fetch-config": "yarn config-manager fetchConfig app-name",
```

Comment:
Update to new syntax for config manager eg https://github.com/reapit/foundations/blob/master/packages/elements/package.json#L32